profiles/profiles.desc

```diff
@@ -0,0 +1,6 @@
+hidden  default/linux/amd64/17.1/selinux                        stable
+hidden  default/linux/amd64/17.1/hardened/selinux               stable
+hidden  default/linux/amd64/17.1/no-multilib/hardened/selinux   stable
+hidden  default/linux/amd64/17.0/x32                            stable
+hidden  default/linux/x86/17.0/selinux                          stable
+hidden  default/linux/x86/17.0/hardened/selinux                 stable
```

Comment:
Could these be `dev` instead of `stable`? Or doesn't that work?

Reply:
Looks like it can be dev. I was trying to match the parent, but I might change it to `exp` instead?

Reply:
`exp` would be even better (I was unaware that `exp` was a thing it could be)

Reply:
Actually doing that might exclude it from `repoman -dx` checks. I'll change it to dev.